### ISSUE TITLE
Materials Design

### ISSUE THREAD

**Issue by joeblew99** [ponzu-cms/ponzu#95]
Google have a new material design lib.
Called Materials Components.
https://github.com/material-components/material-components-web

Demo:
http://material-components-web.appspot.com
Many examples show off the i18n support aspects too.
http://material-components-web.appspot.com/textfield.html 

This is their rewrite of Material Design to fix up the limiting architecture that the first v.ersion had. They reached the law of diminishing returns on the first architecture.

They support i18n better with RTL and LTR for example. Its not as complete as the libs out there yet, but its much much tighter than materials Design Lite, etc now.

I think it will be a good fit for Ponzu, but putting this out here for now for comment.

I used this for a quick and dirty project and it was fast with no road blocks. Really nice

**Comment by nilslice** [comment]
Cool - thank you for keeping an eye on that. I hope the [Materialize CSS](http://materializecss.com/) team will make an easy migration update to the latest version of Google's design, since I use their CSS library for the layout and styling of most of the CMS.

**Comment by joeblew99** [comment]
hope so.

I asked them. Lets see
https://github.com/Dogfalo/materialize/issues/4334

**Comment by joeblew99** [comment]
they rejected the idea. 
even though the page ( https://github.com/material-components/material-components-web )  from google says "Material Components for the web is the successor to Material Design Lite, and has 3 high-level goals:"

I

The demo of their code is getting better:
https://material-components-web.appspot.com/

RTL support:
https://material-components-web.appspot.com/grid-list.html

dialogs:
https://material-components-web.appspot.com/dialog.html
- works properly on mobile and with RTL

---

No Cal yet, which is something we need.

--

frameworks:
https://github.com/material-components/material-components-web/tree/master/framework-examples

**Comment by nilslice** [comment]
That's too bad.. I think it would be a good move for them to work on a migration to follow Material Component's progress..

Further down the road, I would like to make the Ponzu admin dashboard more of a webapp with a nicer javascript UI. I think that would be the best time to consider upgrading the css framework to something that has better support for future-proofing

If you're interested in diving deeper into Ponzu's UI, I need to make the current dashboard mobile-friendly... essentially adding the right column classes (s6, mX, etc) to the various UI elements in the admin dashboard. Would love help from anyone on this!

**Comment by joeblew99** [comment]
Sorry about the late reply.

I have been pretty swamped and not had a chance to work on this.
I did try out  MDC when i raised this issue and it was very easy to get going. So i expect it will be easy sailing to do it.

I tried MDC out on mobile. All of it worked. Fast, handled RTL, i18n. Really impressive.
They still don't have a Cal widget yet though. There is an issue for it but we some reason it's not getting any attention from them. 

Anyway it looks very nice.

As far as frameworks to use with it you have 4 to choose from.
There is a gopherjs one being worked on over at gu-io github repo. It can render server side or client side ( gopherjs) but the MDC work is a low priority. So it's not ready. But just saying in case ..
